File: 백준/Bronze/3052. 나머지/나머지.c
#include <stdio.h>

int main(void) {
	int arr[10];
	int total = 0, num;
	for (int i = 0; i < 10; i++) {
		scanf("%d", &num);
		arr[i] = num % 42;
	}
	for (int i = 0; i < 10; i++) {
		int hap = 0;
		for (int j = i + 1; j < 10; j++) {
			if (arr[i] == arr[j]) hap++;
		}
		if (hap == 0) total++;
	}
	printf("%d", total);
	return 0; 
}

File: 백준/Bronze/3052. 나머지/나머지.py
arr = []
total = 0
for i in range(10):
    n = int(input())
    arr.append(n%42)
for i in range(10):
    hap = 0
    for j in range(i + 1, 10):
        if(arr[i] == arr[j]): hap += 1
    if(hap == 0):
        total += 1
print(total)